modules/core/src/main/scala/org/scalasteward/core/vcs/package.scala

```diff
@@ -64,4 +64,21 @@ package object vcs {
     else
       List.empty
   }
+
+  def possibleReleaseNoteFiles(repoUrl: String, update: Update): List[String] = {
+    val canonicalized = repoUrl.replaceAll("/$", "")
+    val vcsSpecific =
+      if (repoUrl.startsWith("https://github.com/"))
```

Comment:
Gitlab offers "Releases" page.
But unlike GitHub, it have no permalink for specific version, and hard to detect since it returns `200` even Releases are empty (not used).

Example
* Releases https://gitlab.com/gitlab-org/gitlab-runner/-/releases
* Empty releases https://gitlab.com/gitlab-org/gitlab-ce/-/releases

Maybe use GitLab's Releases API to detect release (Ref. https://github.com/fthomas/scala-steward/issues/839)

Reply:
I agree that what is in this PR is good enough. Using the GitLab API to find out if a project uses GitLab's releases feature seems to be overkill to me.